generate/unix/Makefile.config

```diff
@@ -148,6 +148,15 @@ ACPICA_HEADERS = \
 OPT_CFLAGS ?= $(CWARNINGFLAGS)
 
 #
+# Common compiler flags
+# The _GNU_SOURCE symbol is required for many hosts.
+#
+ifeq ($(M32),TRUE)
+CFLAGS +=-m32
+LDFLAGS +=-m32
+endif
```

Comment:
I tried calling removing LDFLAGS and calling make M32=TRUE. It turns out that the ACPICA Makefiles do not derive LDFLAGS from CFLAGS. Removing LDFLAGS results in an error when calling make.